base/qaqc/R/find_formats_without_inputs.R

```diff
@@ -13,53 +15,36 @@
 ##' or look at the README
 
 
-find_formats_without_inputs <- function(con, user_id = NULL, created_after = NULL, updated_after = NULL){
+find_formats_without_inputs <- function(con, user_id_code = NULL, created_after = NULL, updated_after = NULL, created_before = NULL, updated_before = NULL){
```

Comment:
addition of "code" to user_id seems superfluous. Just update the documentation to make sure it's clear that this is the _numeric_ id not the username.